sp800-63b/sec5_authenticators.md

```diff
@@ -359,7 +359,7 @@ Biometric matching SHOULD be performed locally on claimant's device or MAY be pe
 
 If matching is performed centrally:
 
-* Use of the biometric SHALL be bound tightly to a single, specific device that is identified using approved cryptography.
+* Use of the biometric SHALL be limited to a single, specific device that is identified using approved cryptography.
```

Comment:
I'm not sure that works with the next bullet, which requires use of templates and biometric revocation. The idea is that you enroll and use the biometric with a specific authenticator -- the server should maintain a separate template for each device, somewhat analogous to a hash of a password. But might want to confirm with @enewt .